s2spy/_resample.py

```diff
@@ -219,7 +224,7 @@ def resample(
         raise ValueError("Generate a calendar map before calling resample")
 
     utils.check_timeseries(input_data)
-    utils.check_input_frequency(mapped_calendar, input_data)
+    #utils.check_input_frequency(mapped_calendar, input_data)
```

Comment:
Is there a reason to leave commented code in the PR? Why not remove it?

Reply:
Because this check is still valid for all calendars with the argument `freq`, but not for `CustomCalendar`. When we finish rebasing everything on `CustomCalendar`, we need to bring it back.

But good comment! Now I add a more explicit note there, just to remind ourselves:

```py
    # This check is still valid for all calendars with `freq`, but not for CustomCalendar
    # TO DO: add this check when all calendars are rebased on the CustomCalendar
    #utils.check_input_frequency(mapped_calendar, input_data)
```